app/views/devise/confirmations/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class='row'>
+  <div class='col-md-6'>
+    <h3>Please enter your password</h3>
+    <%= form_for(resource, :as => resource_name, :url => confirm_with_password_path(resource_name), :html => { :method => :put, :class => 'form' }) do |f| %>
+      <%= resource_error_messages!(resource) %>
+
+      <div class='form-group'>
+        <%#= f.label :email %>
+        <%= f.hidden_field :confirmation_token, :value => resource.confirmation_token %>
+      </div>
+
+      <div class='form-group'>
+        <%= f.label :password %>
```

Comment:
Can we add a hint asking the user to enter the password used at the time of registration? I think formtastic helpers support `hint: '....'`

Reply:
`hint:` didn't do anything on label. Added `placeholder` on password field.